v2/api/sql/v1beta20211101/server_types_gen.go

```diff
@@ -0,0 +1,2817 @@
+	} else {
+		server.FederatedClientId = nil
+	}
+
+	// Identity
+	if source.Identity != nil {
+		var identity ResourceIdentity
+		err := identity.Initialize_From_ResourceIdentity_STATUS(source.Identity)
+		if err != nil {
+			return errors.Wrap(err, "calling Initialize_From_ResourceIdentity_STATUS() to populate field Identity")
+		}
+		server.Identity = &identity
+	} else {
+		server.Identity = nil
+	}
+
+	// KeyId
+	server.KeyId = genruntime.ClonePointerToString(source.KeyId)
+
+	// Location
+	server.Location = genruntime.ClonePointerToString(source.Location)
+
+	// MinimalTlsVersion
+	server.MinimalTlsVersion = genruntime.ClonePointerToString(source.MinimalTlsVersion)
+
+	// PrimaryUserAssignedIdentityReference
+	if source.PrimaryUserAssignedIdentityId != nil {
+		primaryUserAssignedIdentityReference := genruntime.CreateResourceReferenceFromARMID(*source.PrimaryUserAssignedIdentityId)
+		server.PrimaryUserAssignedIdentityReference = &primaryUserAssignedIdentityReference
+	} else {
+		server.PrimaryUserAssignedIdentityReference = nil
+	}
+
+	// PublicNetworkAccess
+	if source.PublicNetworkAccess != nil {
+		publicNetworkAccess := ServerProperties_PublicNetworkAccess(*source.PublicNetworkAccess)
+		server.PublicNetworkAccess = &publicNetworkAccess
+	} else {
+		server.PublicNetworkAccess = nil
+	}
+
+	// RestrictOutboundNetworkAccess
+	if source.RestrictOutboundNetworkAccess != nil {
+		restrictOutboundNetworkAccess := ServerProperties_RestrictOutboundNetworkAccess(*source.RestrictOutboundNetworkAccess)
+		server.RestrictOutboundNetworkAccess = &restrictOutboundNetworkAccess
+	} else {
+		server.RestrictOutboundNetworkAccess = nil
+	}
+
+	// Tags
+	server.Tags = genruntime.CloneMapOfStringToString(source.Tags)
+
+	// Version
+	server.Version = genruntime.ClonePointerToString(source.Version)
+
+	// No error
+	return nil
+}
+
+// OriginalVersion returns the original API version used to create the resource.
+func (server *Server_Spec) OriginalVersion() string {
+	return GroupVersion.Version
+}
+
+// SetAzureName sets the Azure name of the resource
+func (server *Server_Spec) SetAzureName(azureName string) { server.AzureName = azureName }
+
+// An Azure SQL Database server.
+type Server_STATUS struct {
+	// AdministratorLogin: Administrator username for the server. Once created it cannot be changed.
+	AdministratorLogin *string `json:"administratorLogin,omitempty"`
+
+	// Administrators: The Azure Active Directory administrator of the server.
+	Administrators *ServerExternalAdministrator_STATUS `json:"administrators,omitempty"`
+
+	// Conditions: The observed state of the resource
+	Conditions []conditions.Condition `json:"conditions,omitempty"`
+
+	// FederatedClientId: The Client id used for cross tenant CMK scenario
+	FederatedClientId *string `json:"federatedClientId,omitempty"`
+
+	// FullyQualifiedDomainName: The fully qualified domain name of the server.
+	FullyQualifiedDomainName *string `json:"fullyQualifiedDomainName,omitempty"`
```

Comment:
Do we want to export FQDN for this into a configmap?

Reply:
Good catch! Yes I think that makes sense.

Other DBs export this as a secret but that was because we didn't have the auto-configmap export. We can add it to them as well if we want, but it probably makes sense here to use that rather than into a secret.

Reply:
This is fixed